src/schemas/listing-core.json

```diff
@@ -0,0 +1,243 @@
+{
+  "$id": "http://schema.originprotocol.com/listing-core-v1.0.0",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Origin Protocol core listing schema",
+  "definitions": {
+    "moneyAmount": {
+      "type": "object",
+      "title": "An amount of money in a specified currency",
```

Comment:
I'm not totally familiar with the guts of json schema. Will `title` ever be shown to users? If so, it should be replaces with a generic identifier that can be localized, e.g. https://github.com/OriginProtocol/origin-dapp/blob/master/public/schemas/housing.json#L15

Reply:
No, title does not get shown to users. It simply provides a short description of the purpose of the object. See https://json-schema.org/understanding-json-schema/reference/generic.html

Reply:
Ahhh, understood. I didn't realize this was for _validation_. Though should have clicked that we were in the `origin-js` repo! (For others reading, here is [json schema validation spec](https://datatracker.ietf.org/doc/draft-handrews-json-schema-validation/))

> ...DApp could keep using those schemas to get the data from the user at listing creation time, then data gets passed to origin-js, which creates a JSON object from it, ...

Is the idea that origin-js would _transform_ the data coming from our old schemas do fit into this schema?